core/crypto/crypto_store/crypto_store_impl.cpp

```diff
@@ -164,19 +164,63 @@ namespace kagome::crypto {
     }
     auto kp = findEd25519Keypair(KEY_TYPE_LP2P, keys.value().at(0));
     if (kp) {
-      auto &secret_key = kp.value().secret_key;
-      auto &public_key = kp.value().public_key;
-      libp2p::crypto::PublicKey lp2p_public{
-          {libp2p::crypto::Key::Type::Ed25519,
-           std::vector<uint8_t>{public_key.cbegin(), public_key.cend()}}};
-      libp2p::crypto::PrivateKey lp2p_private{
-          {libp2p::crypto::Key::Type::Ed25519,
-           std::vector<uint8_t>{secret_key.cbegin(), secret_key.cend()}}};
-      return libp2p::crypto::KeyPair{
-          .publicKey = lp2p_public,
-          .privateKey = lp2p_private,
-      };
+      return edKeyToLibp2pKeypair(kp.value());
```

Comment:
Name of function looks a bit weird. I suggest to use full name of standard everywhere if it is used.
```suggestion
      return ed25519KeyToLibp2pKeypair(kp.value());
```
or 
```suggestion
      return ed25519ToLibp2pKeypair(kp.value());
```